lib/helper/Nightmare.js

```diff
@@ -1066,6 +1066,28 @@ class Nightmare extends Helper {
     return this.browser.refresh();
   }
 
+  /**
+   * {{> saveElementScreenshot }}
```

Comment:
where is file with ext. `mustache`?

Reply:
Is there a command to generate .mustache files?, or do we have to manually create one

Reply:
Need add to manually. You can see example [here](https://github.com/codeceptjs/CodeceptJS/blob/master/docs/webapi/amOnPage.mustache)

Reply:
And then run `npm run docs`

Reply:
@suniljaiswal01 yes. this is a placeholder for a shared template. Because all standard actions have same documentation, we store them in docs/webapi/*.mustache so when you add `saveElementScreenshot` you are referring to: `docs/webapi/saveElementScreenshot.mustache` file to be included as documentation.

So please add this documentation file and I am ready to merge this PR and make a release

Reply:
mustache file added